src/widgets/main/ui/FormFilter/index.tsx

```
@@ -0,0 +1,71 @@
+'use client';
+
+import { useState } from 'react';
+import { ArrowDown, ArrowUp } from '@/shared/assets/icons';
+import { OptionType } from '@/entities/main/model/types';
```

ESLint

[error] `@/entities/main/model/types` import should occur before import of `@/shared/assets/icons`

(import/order)